toxcore/LAN_discovery.c

```diff
@@ -186,7 +186,7 @@ static void fetch_broadcast_info(uint16_t port)
             continue;
         }
 
-        struct sockaddr_in *sock4 = (struct sockaddr_in *)&i_faces[i].ifr_broadaddr;
+        struct sockaddr_in *sock4 = (struct sockaddr_in *)(void *)&i_faces[i].ifr_broadaddr;
```

Comment:
nothing to do with net_pack

Reply:
True, it's about fixing invalid casts. This cast seems invalid but it's not. The void cast makes it explicit that we know what we're doing.